beacon-chain/rpc/beacon/blocks_test.go

```diff
@@ -152,6 +152,9 @@ func TestServer_ListBlocks_Genesis_MultiBlocks(t *testing.T) {
 }
 
 func TestServer_ListBlocks_Pagination(t *testing.T) {
+	params.UseMinimalConfig()
```

Comment:
These tests should have been minimal to begin. Only reason they were passing is because bazel leaked minimal config to here